### PR TITLE
refac: refactor `TwoAdicFri` verify

### PR DESCRIPTION
**Pull request by chokobole**
# Description

It refactors the code that is used in verfication.

### CODE REVIEW

**Review by batzor — APPROVED** [review]
LGTM

**Review by GideokKim — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

File: tachyon/crypto/commitments/fri/simple_fri.h [hunk 1]
```diff
@@ -153,14 +153,14 @@ class SimpleFRI final
       // Pᵢ₊₁(X) = Pᵢ_even(X) + β * Pᵢ_odd(X)
       //
       // If the domain of Pᵢ(X) is Dᵢ = {ω⁰, ω¹, ..., ωⁿ⁻¹},
-      // then the domain of Pᵢ₊₁(X) is Dᵢ₊₁ = {ω⁰, ω¹, ..., ωᵏ⁻¹},
+      // then the domain of Pᵢ₊₁(X) is Dᵢ₊₁ = {ω⁰, ω², ..., ωⁿ⁻²},
       // where k = n / 2.
```
Comment (ashjeong) [inline tachyon/crypto/commitments/fri/simple_fri.h:157]: Could you remove this line?

**Review by ashjeong — APPROVED** [review]
LGTM

**Pull request merged**